src/geo/transform.js

```diff
@@ -206,10 +206,10 @@ class Transform {
 
     /**
      * Return a zoom level that will cover all tiles the transform
-     * @param {Object} options
-     * @param {number} options.tileSize
-     * @param {boolean} options.roundZoom
-     * @returns {number} zoom level
+     * @param {Object} options options
```

Comment:
There are a lot of `options` params. It might not be worthwhile to add descriptions to each, but I am not sure how to configure eslint to ignore them.

Reply:
Let's see how this looks on the live docs — we might need to come up with a way to filter this out or maybe PR the plugin repo with an option to ignore.